lang/cs/Org.Apache.REEF.IO/PartitionedData/FileSystem/FileSystemInputPartition.cs

```diff
@@ -123,23 +131,23 @@ private ISet<string> Download()
         /// provided by the Serializer
         /// </summary>
         /// <returns></returns>
-        public T GetPartitionHandle()
+        public IEnumerable<T> GetPartitionHandle()
```

Comment:
I agree it is clear. That was the reason I used it at the first place. But Dhruv asked for the change because of IMRU usage as I remembered. Just need to clarify from him.